toolkit/resources/imageconfigs/packagelists/iso-initrd-packages.json

```diff
@@ -51,6 +51,7 @@
         "openssh-server",
         "openssl",
         "mariner-release",
+        "mariner-repos",
```

Comment:
Why are we doing this? What's the scenario/requirement/project? So far the ISO didn't use any external repositories by design, as it was expected to have all of the packages required by the final system available locally.

Reply:
This is for the PXE boot scenario where users won't pull packages from the local RPM repo built into the ISO but from external repos. Thus, these keys would need to be reside within the installer for that operation to succeed.